packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1420,7 +1420,9 @@ class _TabBarViewState extends State<TabBarView> {
     if (widget.controller != oldWidget.controller) {
       _updateTabController();
       _currentIndex = _controller!.index;
+      _warpUnderwayCount++;
```

Comment:
If the view metrics become larger, the scrolling will be divided into two stages, the second stage occurs in the layout stage, which will trigger a notification and then cause an exception.
In fact, we can ignore the scroll notification for the scrolling of this scene.